allies.MD

```diff
@@ -0,0 +1,19 @@
+# Resources for allies
+
+## Why does it matter to you?
+Put shortly:
+- If you advertise jobs - [the wording you use impacts who applies](https://www.glassdoor.com/employers/blog/10-ways-remove-gender-bias-job-listings/) (great candidates might be self-excluding).
+- If you hire candidates - [the techniques you use impact who gets through](https://hbr.org/2017/06/7-practical-ways-to-reduce-bias-in-your-hiring-process) (you might not hire or promote the best candidates).
+- If you intend to retain employees - [the culture of your organisation impacts who wants to stay](https://leanin.org/women-in-the-workplace-report-2022/were-in-the-midst-of-a-great-breakup) (including genuine inclusion of diversity and equitable opportunities).
```

Comment:
When I tried to access this link, it said "This page does not exist", please consider updating it.Thanks